src/Parameters/InsertDocumentParameters.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of littleredbutton/bigbluebutton-api-php.
+ *
+ * littleredbutton/bigbluebutton-api-php is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * littleredbutton/bigbluebutton-api-php is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with littleredbutton/bigbluebutton-api-php. If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace BigBlueButton\Parameters;
+
+/**
+ * @method string getMeetingID()
+ * @method $this setMeetingID(string $id)
+ */
+class InsertDocumentParameters extends MetaParameters
+{
+    /**
+     * @var string
+     */
+    protected $meetingID;
+
+    /**
+     * @var array
+     */
+    protected $presentations = [];
+
+    public function __construct(string $meetingID)
+    {
+        $this->meetingID   = $meetingID;
+    }
+
+    /**
+     * @return CreateMeetingParameters
+     */
+    public function addPresentation(string $url, string $filename, ?bool $downloadable = null, ?bool $removable = null)
+    {
+        $this->presentations[$url] = [
+            'filename'     => $filename,
+            'downloadable' => $downloadable,
+            'removable'    => $removable,
```

Comment:
What about a data object here? ;) (yeah, that's almost internal, but we exposing it also via a getter to the user).

Reply:
Than we should remove the getter :rofl:

Reply:
Would be also fine for me ^^ . I didn't want to propose that directly as I was what is the goal of it ^^ .

Reply:
It's only there, because it's also there in the create meeting params class. I remove it and add a `removePresentation` method, which is much more useful.